<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>60</y>
     <width>772</width>
     <height>407</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>772</width>
      <height>78</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="blue_state">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="blue_state_value">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>45</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="blue_temp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>45</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue_hum">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>45</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>湿度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue_sensor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>116</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>传感器类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue_sensor_value">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>45</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SHT11</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue_temp_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>45</y>
       <width>18</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue_hum_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>45</y>
       <width>15</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>84</y>
      <width>772</width>
      <height>322</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
